signbank/dictionary/models.py

```diff
@@ -630,6 +644,21 @@ def __str__(self):
         return str(self.morpheme.idgloss) + ' is ' + str(self.role) + ' of ' + str(self.parent_gloss.idgloss)
 
 
+@python_2_unicode_compatible
+class Signer(models.Model):
+    """The list of signers"""
+    #: Signer name.
+    name = models.CharField(max_length=150, unique=False)
```

Comment:
Should the signer's name be unique to reduce duplicates?

Reply:
Good suggestion. Will add that.